pkg/vf/virtionet_test.go

```diff
@@ -0,0 +1,100 @@
+package vf
+
+import (
+	"fmt"
+	"math/rand"
+	"net"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/crc-org/vfkit/pkg/config"
+)
+
+func testConnectUnixgram(t *testing.T) error {
+	unixSocketPath := filepath.Join("/tmp", fmt.Sprintf("vnet-test-%x.sock", rand.Int31n(0xffff))) //#nosec G404 -- no need for crypto/rand here
+	addr, err := net.ResolveUnixAddr("unixgram", unixSocketPath)
+	if err != nil {
```

Comment:
In test code, we usuallly use `github.com/stretchr/testify` and `require.NoError(t, err)` which is more concise.

Reply:
Thanks, replaced!! 👍